### ISSUE TITLE
Does this version work with Sketch 39+ ??

### ISSUE THREAD

**Issue by kristianmandrup** [fnky/sketch-devtools-assistant#2]
How do I install it? I tried opening the project in XCode and run it from there, but...

`ld: library not found for -lPods-Sketch DevTools Assistant`

**Comment by fnky** [comment]
No, theres no changes in this repository. The 2 commits were only cleanup. Apart from that, this is no more different than the original repository.
